app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/addedit/VaultAddEditSshKeyItems.kt

```diff
@@ -46,7 +46,8 @@ fun LazyListScope.vaultAddEditSshKeyItems(
         BitwardenTextField(
             label = stringResource(id = R.string.public_key),
             value = sshKeyState.publicKey,
-            onValueChange = sshKeyTypeHandlers.onPublicKeyTextChange,
```

Comment:
Don't we need this stuff for adding a new ssh key?

I would expect something like this:
```kotlin
readOnly = isAddMode
```

Reply:
We will not be manually adding SSH keys. They will be generated by the SDK, but that has not been implemented yet.

Reply:
Ohh, that makes things a bit weirder. Do we have more updates coming for the add screen?

Reply:
Yes, at some point. I've not seen the add flow for mobile yet, though. We only have the filter, listing, view, and edit Figma designs so far. I can speculate based on the desktop client, but I don't want to get ahead of ourselves.

Reply:
OK